docs: Mention good contrast requirement (#32)
ReVanced/revanced-branding: README.md

- Don't change color
- Don't accompany the logo with a headline starting with anything other than the wordmark
- Don't overemphasize the logo

> - Don't place the logo in an area or use variant in an area with low constrast

Reviewer: ```suggestion
- Don't place the logo or a variant on a background with low contrast
```

> - Don't place the logo in an area or use variant in a way that hampered accessibility or have less than 3.5:1 contrast ratio

Author: ```suggestion
- Don't place the logo in an area or use variant in a way that hampered accessibility or have less than 3.1:1 contrast ratio
```
